### PR TITLE
Add Assets Page FE

### PR DESCRIPTION
**Pull request by pivilartisant**
Screenshot 2023-08-03 at 11 17 38
Screenshot 2023-08-03 at 11 17 49

### CODE REVIEW

File: web-frontend/src/pages/Assets/Assets.tsx [hunk 1]
```diff
@@ -1,15 +1,108 @@
...
+
+  const tokenList = [massaToken, mrc20Token, mrc721Token];
+
   return (
     <WalletLayout menuItem={MenuItem.Assets}>
-      <Placeholder
-        message={Intl.t('placeholder.teaser-assets-page')}
-        icon={<FiDisc size={114} />}
-      />
+      <div className="flex flex-col bg-secondary p-10 h-fit w-2/3 rounded-lg">
```
Comment (maxime-carabina) [inline web-frontend/src/pages/Assets/Assets.tsx:51]:
```suggestion
      <div className="bg-secondary p-10 h-fit w-2/3 rounded-lg">
```

I think you don't need to put flex here. Can you test it for me ? 🤔

File: web-frontend/src/pages/Assets/Assets.tsx [hunk 2]
```diff
@@ -1,15 +1,108 @@
...
+        <div className="flex flex-row items-center justify-between w-full mb-6">
```
Comment (maxime-carabina) [inline web-frontend/src/pages/Assets/Assets.tsx:52]:
```suggestion
        <div className="flex items-center justify-between w-full mb-6">
```

When you add `flex` value, you don't need to precise `flex-row` to put item in the same line 😉

File: web-frontend/src/pages/Assets/Assets.tsx [hunk 3]
```diff
@@ -1,15 +1,108 @@
...
+            {Intl.t('assets.import')}
+          </Button>
+        </div>
+        <div className="flex flex-col w-full h-fit bg-primary rounded-lg gap-4 p-8">
+          {tokenList.map((token, index) => (
+            <Token
+              token={token}
+              key={index}
+              // This will change when we have token consts
+              disable={token?.name === 'Massa' ? true : false}
```
Comment (maxime-carabina) [inline web-frontend/src/pages/Assets/Assets.tsx:70]: Why you need to disabled `Token` component when the token name is Massa ? 🤔

File: web-frontend/src/pages/Assets/Assets.tsx [hunk 4]
```diff
@@ -1,15 +1,108 @@
...
+            fullMode={true}
+            onClose={() => setModal(false)}
+          >
+            <PopupModalHeader>
+              <div className="mas-title mb-6">
+                {Intl.t('assets.import-title')}
+              </div>
+            </PopupModalHeader>
+            <PopupModalContent>
+              <div className="flex flex-col mas-body2 pb-10">
```
Comment (maxime-carabina) [inline web-frontend/src/pages/Assets/Assets.tsx:89]:
```suggestion
              <div className="mas-body2 pb-10">
```

**Comment by mazmassa** [comment]
Please, add mirage mock first! :;

File: web-frontend/src/i18n/en_US.json [hunk 5]
```diff
@@ -12,6 +12,13 @@
       "receive": "Receive"
     }
   },
+  "assets": {
```
Comment (mazmassa) [inline web-frontend/src/i18n/en_US.json:15]: Please, add mirage mock first! :;

File: web-frontend/src/pages/Assets/Assets.tsx (hunk 3 above)
Comment (pivilartisant) [reply web-frontend/src/pages/Assets/Assets.tsx:70]: because the user shouldn't be able to delete the massa token

**Comment by longgamma-massa** [comment]
Scope update: @pivilartisant 
A) Balance of XMA (Massa coin) will be available on the list by default (even if no other assets have been added via functionality: "Import token" (by default this page has 1 row for XMA). It will be available in the backend.
B) Changed [the title](https://www.figma.com/file/LSaOibVDFb1reF6FePVn5l/Massastation?type=design&node-id=324-5834&mode=design&t=2XLlQHBxJiui1vW9-0) of the table from 'Tokens' to 'Assets'.

File: .DS_Store
[the remark this answers is not in the document]
Comment (pivilartisant) [reply .DS_Store:1]: I have no idea it comes from the generate i think

[the remark this answers is not in the document]
Comment (maxime-carabina) [reply .DS_Store:1]: Okay! So can you remove it? 😉

**Review by maxime-carabina — APPROVED** [review]

File: .DS_Store
Comment (Thykof) [inline .DS_Store:1]: seems like github is bugging, I can't see this file on my local repo

**Review by mazmassa — APPROVED** [review]
Thanks for that Paul! 🎉

**Pull request merged**